packages/react-components/src/components/Toast/Toast.stories.tsx

```diff
@@ -1,64 +1,86 @@
 import * as React from 'react';
 import { ComponentMeta, Story } from '@storybook/react';
 
-import { ToastProps, Toast as ToastComponent } from './Toast';
+import { StoryDescriptor } from '../../stories/components/StoryDescriptor';
+import { ToastProps, Toast } from './Toast';
 
 export default {
   title: 'Components/Toast',
-  component: ToastComponent,
+  component: Toast,
   parameters: {
     componentSubtitle: `
     Toast is a small message that by default shows up in the top middle of the screen. 
     It disappears on its own after a few seconds. It provides a feedback about an operation 
     for user.
     `,
   },
-} as ComponentMeta<typeof ToastComponent>;
+} as ComponentMeta<typeof Toast>;
 
-interface IToastArgs extends ToastProps {
-  label: string;
-}
-
-const StoryTemplate: Story<IToastArgs> = (args: IToastArgs) => (
-  <div>
-    <ToastComponent {...args}>Example text</ToastComponent>
-  </div>
+export const Default: Story<ToastProps> = (args: ToastProps) => (
+  <Toast {...args}>All systems running</Toast>
 );
+Default.storyName = 'Toast';
+Default.args = {};
 
-export const Toast = StoryTemplate.bind({});
-Toast.args = {};
-
-export const SuccessToast = StoryTemplate.bind({});
-SuccessToast.args = {
-  kind: 'success',
-};
-
-export const WarningToast = StoryTemplate.bind({});
-WarningToast.args = {
-  kind: 'warning',
-};
-
-export const ErrorToast = StoryTemplate.bind({});
-ErrorToast.args = {
-  kind: 'error',
-};
-
-export const InfoToast = StoryTemplate.bind({});
-InfoToast.args = {
-  kind: 'info',
-};
+export const Kinds = (): JSX.Element => (
+  <>
+    <StoryDescriptor title="Success">
+      <Toast kind="success">Saved successfully</Toast>
+    </StoryDescriptor>
+    <StoryDescriptor title="Error">
+      <Toast kind="error">System error</Toast>
+    </StoryDescriptor>
+    <StoryDescriptor title="Warning">
+      <Toast kind="warning">Check your system</Toast>
+    </StoryDescriptor>
+    <StoryDescriptor title="Info">
+      <Toast kind="info">All systems running</Toast>
+    </StoryDescriptor>
+  </>
+);
 
-export const NotificationToast = StoryTemplate.bind({});
-NotificationToast.args = {
-  kind: 'notification',
-};
+export const WithCloseIcon = (): JSX.Element => (
+  <Toast removable={true} onClose={() => alert('Close icon clicked')}>
+    All systems running
+  </Toast>
+);
 
-export const ToastWithActionAndRemovable = StoryTemplate.bind({});
-ToastWithActionAndRemovable.args = {
-  action: {
-    label: 'Action',
-    handler: () => alert('onAction click'),
-  },
-  removable: true,
-  onClose: () => alert('onClose click'),
-};
+export const WithCustomAction = (): JSX.Element => (
+  <>
+    <StoryDescriptor title="Without close icon">
+      <Toast
+        action={{
+          label: 'Show details',
+          onClick: () => alert('Custom action button clicked'),
```

Comment:
![image](https://user-images.githubusercontent.com/23378800/183626880-0b4d98a6-92bd-48da-a7f6-077df144983c.png)

We can use the SB functionality to track such events in UI (Story > Canvas > Actions)

Example from `Picker` component stories:
`export default {
  title: 'Components/Picker',
  component: PickerComponent,
  parameters: {
    componentSubtitle: `TBD`,
  },
  argTypes: {
    onSelect: { action: 'changed' },
  },
} as ComponentMeta<typeof PickerComponent>;`

Reply:
Ok, I think I should do so in the main story, as in the remaining ones I use regular components. I'll think about how to handle it in this specific case.

Reply:
I've applied some changes e2b6981